editor/scene_tree_dock.cpp

```diff
@@ -3453,6 +3465,8 @@ void SceneTreeDock::_tree_rmb(const Vector2 &p_menu_pos) {
 				bool placeholder = selection[0]->get_scene_instance_load_placeholder();
 				if (profile_allow_editing) {
 					menu->add_check_item(TTR("Editable Children"), TOOL_SCENE_EDITABLE_CHILDREN);
+					menu->set_item_shortcut(menu->get_item_idx_from_text(TTR("Editable Children")), ED_GET_SHORTCUT("scene_tree/toggle_editable_children"));
```

Comment:
```suggestion
					menu->set_item_shortcut(-1, ED_GET_SHORTCUT("scene_tree/toggle_editable_children"));
```
This will use last added item.

Reply:
Fixed.